packages/playground/remote/project.json

```diff
@@ -53,9 +52,18 @@
 		"publish": {
 			"executor": "nx:run-commands",
 			"options": {
-				"command": "node tools/scripts/publish.mjs playground {args.ver} {args.tag}",
+				"command": "node tools/scripts/publish.mjs playground-remote {args.ver} {args.tag}",
 				"parallel": false
 			},
+			"dependsOn": ["build", "build:README-npm"]
+		},
+		"build:README-npm": {
+			"executor": "nx:run-commands",
+			"options": {
+				"commands": [
+					"cp packages/playground/remote/README-npm.md dist/packages/playground/remote/README.md"
```

Comment:
I opted for creating a `README-npm.md` file that is specifically for the published package. The reasoning for this was that, in the future, we might want to add more information to the main `README.md` that is not relevant to the published package. Please let me know if this doesn't make sense, and I'll be happy to adapt as needed.

Note that `README-npm.md` gets renamed to just `README.md` in the published package.

Reply:
I'm not sure. The current README doesn't make sense as an NPM readme, but I don't like the idea of maintaining two. 

I'm wondering how other projects resolve this.

Reply:
I agree that maintaining two READMEs might not be the best approach.

Would it be acceptable to address that problem in #1925? If so, I would remove the `README-npm` bits from here, and there would be no README in the published `@wp-playground/remote` package. We would then fix all the READMEs in #1925.

Reply:
> Would it be acceptable to address that problem in https://github.com/WordPress/wordpress-playground/issues/1925? 

Sounds good. Thank you for working on the README.